mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -166,6 +166,74 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 							ForceNew:    true,
 							Description: `The image from which to initialize this disk. This can be one of: the image's self_link, projects/{project}/global/images/{image}, projects/{project}/global/images/family/{family}, global/images/{image}, global/images/family/{family}, family/{family}, {project}/{family}, {project}/{image}, {family}, or {image}. ~> Note: Either source or source_image is required when creating a new instance except for when creating a local SSD.`,
 						},
+						"source_image_encryption_key": {
```

Comment:
Given this is a handwritten file, you'll need to edit the website file by hand to add documentation.

Reply:
Done